pkg/scanner/local/scan.go

```diff
@@ -174,6 +181,21 @@ func (s Scanner) scanLibrary(apps []ftypes.Application, listAllPackages bool) (r
 	return results, nil
 }
 
+func skipped(filePath string, skipDirectories []string) bool {
+	for _, skipDir := range skipDirectories {
+		skipDir = strings.TrimLeft(filepath.Clean(skipDir), string(os.PathSeparator))
+		rel, err := filepath.Rel(skipDir, filePath)
+		if err != nil {
+			log.Logger.Warn(err)
```

Comment:
I'm wondering when we can get this warning? It seem that Rel() always returns a path prefixed with `..`
Even if it happens we might want to add a message to give some context

Reply:
You're right. `Rel()` returns an error if `skipDir` is an absolute path and `filePath` is a relative path and vice versa. But the both don't have the leading slash in this case. At first, I just suppressed this error, but I added the error handling just in case. It is not supposed to reach this warning. Do you think we should ignore this error like `rel, _ := filepath.Rel(skipDir, filePath)`? Or, should we give more context even if it will be never called? However, it is possible that future changes of `Rel()` implementation may cause a new error.

Reply:
The law of IT says: If it's not supposed to happen, sooner or later it's gonna happen :) That said I'd add contextual message to the warning, e.g.: `log.Logger.Warnf("Unexpected error: %v", err)`

Reply:
done